### ISSUE TITLE
check validations without submitting the form

### ISSUE THREAD

**Issue by salmanxoxo** [NewOldMax/react-material-ui-form-validator#170]
i have two forms in a page for the user to fill. user go to next form only after filling all fields in the first form. to check if user has completely filled first form i am using `this.form.walk(this.form.childs)`. this is not working properly. even if there are not error shown in the form `this.form.errors` will have the last error that was shown in the form. 

also if i keep on submitting form with errors in it `this.form.errors` will have duplicate of the same errors and when i do `this.form.walk` these errors will be there in `this.form.errors`. 

im checking `this.form.errors.length` to check if there are any errors in the form. if not i should take the user to the next form.
how can i achieve my case?

**Comment by NewOldMax** [comment]
@salmanxoxo Hi
did you try to use [isFormValid ](https://github.com/NewOldMax/react-form-validator-core#validatorform) method?

**Comment by salmanxoxo** [comment]
@NewOldMax  yes. isFormValid shows true only if there are any errors that populate after form.submit is called.

**Comment by NewOldMax** [comment]
https://newoldmax.github.io/react-material-ui-form-validator/
there is a "Step-by-step" example where user can't go to next form without completing current one
